gen.treesitter/src/main/java/com/github/gumtreediff/gen/treesitter/AbstractTreeSitterGenerator.java

```diff
@@ -50,6 +63,25 @@ protected TreeContext generate(Reader r) throws IOException {
 
     @Override
     protected String[] getCommandLine(String file) {
-        return new String[]{TREESITTER_CMD, file, getParserName()};
+        if (!TREESITTER_CMD.endsWith(".py")) {
+            return new String[]{TREESITTER_CMD, file, getParserName()};
+        } else {
+            if (CUSTOM_PYTHON_PATH != null) {
+                return new String[]{CUSTOM_PYTHON_PATH, TREESITTER_CMD, file, getParserName()};
+            } else {
+                if (SYSTEM_TYPE.startsWith("windows")) {
+                    return new String[]{PYTHON, TREESITTER_CMD, file, getParserName()};
```

Comment:
I think we can launch this command regardless of OS if you use a default for `PYTHON_CMD`, do you think it can fail in some case? The only "strange" thing is that we need to have `tree-sitter-parser.py` in the PATH.

Reply:
I think you're right, it's that I'm overly complicated🤯

Reply:
However, we can't set the default value of `PYTHON_CMD` to `"python"`, because `python3` is the available Python interpreter in most Linux applications